lib/load.php

```diff
@@ -76,16 +79,19 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/utils.php';
 require __DIR__ . '/editor-settings.php';
 
-require __DIR__ . '/full-site-editing.php';
+if ( ! class_exists( 'WP_Block_Template ' ) ) {
+	require __DIR__ . '/full-site-editing/class-wp-block-template.php';
+}
+require __DIR__ . '/full-site-editing/full-site-editing.php';
```

Comment:
We'd need to better organize these files in core

Reply:
yes, even in the plugin, it deserves a follow-up. What I did in this PR is just gather them in a single folder as they were a bit scattered.